packages/docs/docs/guide/markdown.md

```diff
@@ -8,44 +8,39 @@ Headers automatically get anchor links applied. Rendering of anchors can be conf
 
 ### Internal Links
 
-Inbound links ending in `.md` or `.html` are converted to `<router-link>` for SPA navigation.
+Internal links are converted to `<router-link>` for SPA navigation. Also, every `README.md` or `index.md` contained in each sub-directory will automatically be converted to `index.html`, with corresponding url `/`.
 
-Each sub-directory in your static site should contain a `README.md`. It will automatically be converted to `index.html`.
+Given the following directory structure (and providing you are in `foo/one.md`):
 
-::: tip
-When writing the relative path to a directory's `index.html`, don't forget to close it off with a `/`, otherwise you will get a 404. For example, use `/config/` instead of `/config`.
-:::
-
-If you want to link to another markdown file within a directory, remember to:
-
-1.  Append it with either `.html` or `.md`
-2.  Make sure the case matches since the path is case-sensitive
-
-#### Example
-
-Given the following directory structure:
-
-```
+::: files
 .
-├─ README.md
-├─ foo
-│  ├─ README.md
-│  ├─ one.md
-│  └─ two.md
-└─ bar
-   ├─ README.md
-   ├─ three.md
-   └─ four.md
-```
+├── README.md
+├── foo
+│   ├── README.md
+│   ├── one.md
+│   └── two.md
+└── bar
+&nbsp;   ├── README.md
+&nbsp;   ├── three.md
+&nbsp;   └── four.md
```

Comment:
Same here, and please remove `&nbsp;` here.

BTW, I don't think `files` container should be used in any other location except for two parts of the directory structure.

Reply:
There are actually more than two parts that show directory structure.